src/views/pages/admin.ejs

```diff
@@ -6,11 +6,12 @@
   <title>GUH18 Hub - Admin</title>
   <link rel="stylesheet" href="/css/admin.css">
   <script src="/js/hardwareAdmin.js"></script>
+  <script src="/js/pushNotification.js"></script>
 </head>
 
 <body>
   <div class="wrapper">
-    <% include ../partials/navbar.ejs %>
+    <%- include('../partials/navbar.ejs', {userIsOrganiser: userIsOrganiser || undefined}) %>
```

Comment:
You should write {userIsOrganiser: locals.userIsOrganiser || undefined} instead because accessing userIsOrganiser directly will throw an error if it's not attached to the model.

Reply:
Also is there any reason why `{userIsOrganiser: locals.userIsOrganiser}` wouldn't work?

Reply:
> Also is there any reason why `{userIsOrganiser: locals.userIsOrganiser}` wouldn't work?

That would work too, I'm not sure which one is better
I suppose your latest suggestion is cleaner

Reply:
Changed in the latest commit